MYshort.bib

```diff
@@ -353,6 +353,7 @@ @String{PPL
 @String{PNAS = "PNAS"}
 @String{QREI = "QREI"}
 @String{REJ = "REJ"}
+@String{SA = "Sci. Adv."}
```

Comment:
This abbreviation is inconsistent with our other abbreviations (which for quite some journals are actually not the official ones as documented for JSME below for example). So I guess sticking to the actual abbreviation here makes sense. So I think there is need for changes here but maybe we should open an Issue here? Maybe having a third MYxxxx.bib for real journal abbreviations could make sense? :thinking: I am undecided on how to proceed here but it should probably be made consistent somehown.

Reply:
I changed this to SciAdv, is this okay?

Reply:
I think that is a good compromise for now. :) Thanks for opening the respective issue!